frontends/ol-components/src/components/LearningResourceExpanded/InfoSectionV2.tsx

```diff
@@ -392,11 +513,14 @@ const InfoSectionV2 = ({ resource }: { resource?: LearningResource }) => {
   }
 
   return (
-    <InfoItems data-testid="drawer-info-items">
-      {infoItems.map((props, index) => (
-        <InfoItem key={index} {...props} />
-      ))}
-    </InfoItems>
+    <>
+      <DifferingRunsTable resource={resource} />
```

Comment:
Minor suggestion to pull `<DifferingRunsTable>` to a new file and import it to LearningResourceExpandedV2.tsx as it doesn't use anything in InfoSectionV2.tsx.